test/functional/feature_llmq_dkg_intake.py

```
@@ -68,15 +97,37 @@ def get_score():
         #   discouragement threshold, so banscore stays observable for the score==100 cases.
-        # -debug=net surfaces the Misbehaving reason strings in debug.log.
-        extra_args = [["-whitelist=127.0.0.1", "-debug=net", "-deprecatedrpc=banscore"]] * 4
+        # -debug=net surfaces the Misbehaving reason strings in debug.log, while
+        # -debug=llmq-dkg exposes worker and queue-boundary behavior.
+        common_args = [
+            "-whitelist=127.0.0.1",
+            "-debug=net",
+            "-debug=llmq-dkg",
+            "-deprecatedrpc=banscore",
+        ]
+        extra_args = [common_args + ["-watchquorums=1"]] + [common_args] * 3
```

Ruff (0.16.1)

[warning] Consider `[*common_args, "-watchquorums=1"]` instead of concatenation

Replace with `[*common_args, "-watchquorums=1"]`

(RUF005)